packages/select-rich/src/LionSelectInvoker.js

```diff
@@ -15,9 +15,17 @@ export class LionSelectInvoker extends LionButton {
     return [
       super.styles,
       css`
+        :host {
+          justify-content: space-between;
+        }
+
         #content-wrapper {
           position: relative;
         }
+
+        :host ::slotted([slot='after']) {
```

Comment:
For me this doesn't have any effect. Same for align-items on the host. I guess this is because the button (invoker) takes the height of its contents, and the contents happen to be roughly the same height (arrow / font). Maybe it's better if we leave this CSS out, and leave it up to our subclassers to ensure their rich select invoker looks correct?